### ISSUE TITLE
Возможно стоит декомпозировать этот сервис на несколько других с единой ответственностью

### ISSUE THREAD

**Issue by vladsolomon** [senseiks/cordova-angular-test#2]
https://github.com/senseiks/cordova-angular-test/blob/a7da02f34aa22c819ecb077fdc20ff5947d4bc99/src/app/core/app.service.ts#L34

**Comment by senseiks** [comment]
Да, можно сделать сервис для работы с REST, сервис для работы с локальной
базой и сервис для работы с модулем пользователей.
